InternshipProject#2/Manager/ICommentManager.cs

```diff
@@ -5,9 +5,9 @@ namespace InternshipProject_2.Manager
 {
     public interface ICommentManager
     {
-        //public Task<IEnumerable<Comment>> GetComments(int TicketId);
+        public Task<IEnumerable<Comment>> GetComments(int TicketId);
         public Task CreateComment(CommentRequest newComment);
-        //public Task EditComment(CommentEditRequest editComment);
+        public Task EditComment(CommentEditRequest editComment);
         //public Task DeleteComment(int CommentId);
```

Comment:
You should have only one delete operation and specify the deletion criteria in the request.